docs/OV_Converter_UG/prepare_model/convert_model/Convert_Model_From_PyTorch.md

```diff
@@ -0,0 +1,136 @@
+
+Value for ``example_input`` parameter can be easily derived from the knowledge of input tensor element type and shape. Not always but quite frequently, random numbers suit well for this purpose:
+
+.. code-block:: py
+   :force:
+
+   import torchvision
+   import torch
+   import openvino as ov
+
+   model = torchvision.models.resnet50(pretrained=True)
+   ov_model = ov.convert_model(model, example_input=example_input=torch.rand(1, 3, 224, 224))
+
+In practice, the code to evaluate or test the PyTorch model is usually provided with the model itself and can be used to generate a proper ``example_input`` value. A modified example of using ``resnet50`` model from ``torchvision`` is presented below. It demonstrates how to switch inference in the existing PyTorch application to OpenVINO and how to get value for ``example_input``:
+
+.. code-block:: py
+   :force:
+
+      from torchvision.io import read_image
+      from torchvision.models import resnet50, ResNet50_Weights
+      import requests, PIL, io, torch
+
+      # Get a picture of a cat from the web:
+      img = PIL.Image.open(io.BytesIO(requests.get("https://placekitten.com/200/300").content))
+
+      # Torchvision model and input data preparation from https://pytorch.org/vision/stable/models.html
+
+      weights = ResNet50_Weights.DEFAULT
+      model = resnet50(weights=weights)
+      model.eval()
+      preprocess = weights.transforms()
+      batch = preprocess(img).unsqueeze(0)
+
+      # PyTorch model inference and post-processing
+
+      prediction = model(batch).squeeze(0).softmax(0)
+      class_id = prediction.argmax().item()
+      score = prediction[class_id].item()
+      category_name = weights.meta["categories"][class_id]
+      print(f"{category_name}: {100 * score:.1f}% (with PyTorch)")
+
+      # OpenVINO model preparation and inference with the same post-processing
+
+      import openvino as ov
+      compiled_model = ov.compile_model(ov.convert_model(model, example_input=batch))
+
+      prediction = torch.tensor(compiled_model(batch)[0]).squeeze(0).softmax(0)
+      class_id = prediction.argmax().item()
+      score = prediction[class_id].item()
+      category_name = weights.meta["categories"][class_id]
+      print(f"{category_name}: {100 * score:.1f}% (with OpenVINO)")
+
+**FIXME: Refer to more examples in the notebooks**
```

Comment:
@eaidova Could you help with what to refer here?

Reply:
Added the links.